src/docs/testing/errors.md

```diff
@@ -84,9 +90,45 @@ class MyApp extends StatelessWidget {
 }
 ```
 
+## Set up a Zone errors handling
+
+Consider an `onPressed` callback that invokes an asynchronous function,
+such as `MethodChannel.invokeMethod` (or pretty much any plugin):
+<!-- skip -->
+```dart
+OutlinedButton(
+  child: Text('Click me!'),
+  onPressed: () async {
+    final channel = const MethodChannel('crashy-custom-channel');
+    await channel.invokeMethod('blah');
+  },
+),
+```
+If `invokeMethod` throws an error, it will not be forwarded to `FlutterError.onError`.
+Instead, it will be forwarded to the `Zone` where `runApp` was run.
+
+To catch such an error, use [`runZonedGuarded`][].
+<!-- skip -->
+```dart
+import 'dart:async';
+
+void main() {
+  runZonedGuarded(() async {
+    WidgetsFlutterBinding.ensureInitialized();
+    runApp(MyApp()));
+  }, (Object error, StackTrace stack) {
+    myBackend.sendError(error, stack);
+  });
+}
+```
+Please note that `WidgetsFlutterBinding.ensureInitialized()` is
+called within `runZonedGuarded` - error handling wouldn't work if
+the function would be called from the outside.
+
 [`ErrorWidget.builder`]: {{site.api}}/flutter/widgets/ErrorWidget/builder.html
 [`FlutterError.onError`]: {{site.api}}/flutter/foundation/FlutterError/onError.html
 [`FlutterError.dumpErrorToConsole`]: {{site.api}}/flutter/foundation/FlutterError/dumpErrorToConsole.html
 [`kReleaseMode`]:  {{site.api}}/flutter/foundation/kReleaseMode-constant.html
 [`MaterialApp.builder`]: {{site.api}}/flutter/material/MaterialApp/builder.html
 [reporting errors to a service]: /docs/cookbook/maintenance/error-reporting
+[`runZonedGuarded`]: {site.api}}/flutter/dart-async/runZonedGuarded.html
```

Comment:
Add a [`Zone`] link to this section.

Also, the `runZonedGuarded` link is causing the build failure. It should be ``{{site.api}}``, not `{site.api}``.